pkgs/applications/science/logic/nusmv/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "NuSMV";
+  version = "2.6.0";
+
+  src = fetchurl {
+    url = "https://nusmv.fbk.eu/distrib/NuSMV-${version}-linux64.tar.gz";
```

Comment:
There is no source available?
Or the source is much hard to compile?

Reply:
The source ships with a bunch of patched dependencies and no clear (for me) build instructions so I preferred to patch the binaries.